module/PiCamera/video_resolution.py

```diff
@@ -0,0 +1,75 @@
+class VideoResolution:
+    """The VideoResolution class is responsible for storing a video resolution option
+    As is described on the wiki, the rolling shutter introduces a series of challenges.
+    A rolling shutter can prevent the camera from recording or taking a sequence of pictures with pauses in between.
+    The VideoResolution class will store a singular option of a resolution.
+    """
+
+    def __init__(self, **kwargs):
+        """
+        The constructor instantiate a series of variables that are assigned via key value.
+        Checks will need to be introduced individually
+        :param kwargs: key value arguments that are set via setattr()
+        """
+
+        # Initialising the default expected variables
+        self.width = None
+        self.height = None
+        self.aspect_frame_rate_min = None
+        self.aspect_frame_rate_max = None
+
+        for key, value in kwargs.items():
+            setattr(self, key, value)
+
+    def is_resolution(self, width, height):
+        """
+        Validation if the param
+        :param width: the resolutions width (e.g. 1920)
+        :param height: the resolution height (e.g. 1080)
+        :return:
+        """
+        if self.width is width and self.height is height:
+            return True
+        return False
```

Comment:
Could this be be "return self.width is width and self.height is height"?

Reply:
Indeed, fixed.